src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -47,10 +61,131 @@ public partial class MediaManager : Java.Lang.Object, IPlayer.IListener
 			ControllerAutoShow = false,
 			LayoutParameters = new FrameLayout.LayoutParams(ViewGroup.LayoutParams.MatchParent, ViewGroup.LayoutParams.MatchParent, GravityFlags.CenterHorizontal)
 		};
-
+		Task Result = CheckAndRequestForeGroundPermission(CancellationToken.None);
+		if(Result.IsFaulted)
+		{
+			Logger?.LogError(Result.Exception, "Failed to get Foreground Permission.");
```

Comment:
I don't think we do have a `Logger`, you should use the [Trace.WriteLine](https://learn.microsoft.com/pt-br/dotnet/api/system.diagnostics.trace.writeline?view=net-8.0) instead